bin/publish

```diff
@@ -42,5 +42,17 @@ for image_name in "${IMAGES[@]}"; do
       docker tag "$image_name:$FULL_VERSION_TAG" "$REGISTRY/$image_name:$tag"
       docker push "$REGISTRY/$image_name:$tag"
     done
+
+    for tag in "${TAGS[@]}"; do
+      # Publish only latest to Redhat Registries
```

Comment:
there is an image with a literal `latest` tag, so this comment could be clearer (I don't think you mean that we publish only the `latest` tag to redhat, since you're looping over the `TAGS` array)

Reply:
fwiw, I think it's fine if redhat only has the VERSION tag (and not the `latest`) - I think that's what we do for the authn-k8s client now, and it might be all we can do bc I'm pretty sure RH doesn't let us overwrite tags

Reply:
Ok that definitely simplifies things! I'll just push the latest version, but not do the whole separate thing for the latest tag. I think openshift actually handles that on its own.